src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline

```diff
@@ -250,28 +250,14 @@ module.exports.default = Object.assign({}, module.exports);
 
 ============== file: src/samples/v1/discuss_book_request_streaming_bidi_prog.js ==============
```

Comment:
@alexander-fenster 
Currently the samples live in the gapic source code. Do you think it makes sense to pull them out to the same level as `src`? i.e., `artman-genfiles/language/samples/v1/...`

Reply:
Can we follow the pattern established by existing libraries? That is: https://github.com/googleapis/nodejs-text-to-speech/tree/master/samples

Here `samples` folder is on the top level, it's a sibling to `src`.

Reply:
Sounds good!